samples/sample_common/include/avc_spl.h

```diff
@@ -22,9 +22,13 @@ or https://software.intel.com/en-us/media-client-solutions-support.
 
 #include <vector>
 #include <list>
+#ifndef __clang__
```

Comment:
have you tried to completely remove this pragma and build under Windows? Do we still have any warning?

Reply:
will try